react/features/base/connection/reducer.js

```diff
@@ -9,7 +9,8 @@ import {
     CONNECTION_ESTABLISHED,
     CONNECTION_FAILED,
     CONNECTION_WILL_CONNECT,
-    SET_LOCATION_URL
+    SET_LOCATION_URL,
+    SET_PARENT_LOCATION
```

Comment:
can you put this on top of the list, to avoid conflicts in the future upstream updates